<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransformWidget</class>
 <widget class="QWidget" name="TransformWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>760</width>
     <height>450</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tabUpload">
    <attribute name="title">
     <string>上传列表</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>750</width>
       <height>420</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="upload_scroll">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>748</width>
        <height>418</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tabDownload">
    <attribute name="title">
     <string>下载列表</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>750</width>
       <height>420</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="download_scroll">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>748</width>
        <height>418</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tabTransform">
    <attribute name="title">
     <string>传输记录</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea_3">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>0</y>
       <width>771</width>
       <height>431</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollArea_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>769</width>
        <height>429</height>
       </rect>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>730</width>
         <height>410</height>
        </rect>
       </property>
       <widget class="QTextEdit" name="record_mg">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>50</y>
          <width>671</width>
          <height>301</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>20</y>
          <width>591</width>
          <height>23</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton">
           <property name="text">
            <string>清除记录</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
